test/client/Query.test.tsx

```diff
@@ -163,12 +171,12 @@ describe('Query component', () => {
       const Component = () => (
         <Query query={allPeopleQuery}>
           {result => {
-            if (result.loading) {
+            if (!result || result.loading) {
```

Comment:
I think this is the behavior that most concerns me. I'd like to see this still be an object with thinks like `client`, the observable query methods, and loading set to false (since its skipped its not loading). The only thing that would be missing on a skip would be data / error

Reply:
yeah, this is what I was referring to in my original comment. Should we be hard-coding `data` to `{}` and `loading` to `false`? or remove the `data` field entirely?